src/components/projects.module.css

```diff
@@ -181,12 +258,20 @@
 }
 
 .image-viewer-hero-description {
-  font-size: 1.75vw;
+  font-size: 1.15rem;
```

Comment:
I would reverse this so it's a consistent "mobile-first" approach with:

```
.image-viewer-hero-description {
  font-size: .75rem;
  font-family: var(--neue-family);
  
  @media (min-width: 600px) {
    font-size: 1rem;
  }
    
  @media (min-width: 800px) {
    font-size: 1.15rem;
  }
}
```

Reply:
Thanks! This makes sense.

Reply:
I do get this error when I try this method: 'Expected "context" media feature range notation (media-feature-range-notation)'

Reply:
Ah, it looks like that's coming from https://stylelint.io/user-guide/rules/media-feature-range-notation/. You would need to change it from "context" to "prefix" I guess. It says it can be "limiting" due to https://drafts.csswg.org/mediaqueries/#mq-min-max, but this sort of seems sort of edge case to me. I'm good with way as it's you're project as long as you do mobile first.